ui/src/components/data-entities/customWidgetFields/NestedApiField.js

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+
+import { useDispatch, useSelector } from "react-redux";
+import { useEffect } from 'react';
+import Autocomplete from '@material-ui/lab/Autocomplete';
+import TextField from '@material-ui/core/TextField';
+import pluralize from 'pluralize';
+import {Typography} from "@material-ui/core";
+import {selectedItemsRequested, setNestedField} from "../middleware/entities";
+import {markupProjectionQuery} from "../../utils/helpers";
+
+
+const NestedApiField = (props) => {
+
+  let editItemValues = useSelector(state => state.form.editItem);
+  const dispatch = useDispatch();
+
+  const entity = props.name ;
+  const pluralEntity = pluralize(entity);
+  //const entities =  pluralEntity.charAt(0).toLowerCase() + pluralEntity.slice(1);
+
+  let itemsList = (editItemValues[pluralEntity]) ?editItemValues[pluralEntity][pluralEntity] : [];
+  let selectedItems = (editItemValues[entity + "Selected"]) ? [editItemValues[entity + "Selected"]].filter(Boolean) : [];
+
```

Comment:
this need to be verified if we need to deal with multiple choice. IMO we don't

Reply:
The code can handle at no cost